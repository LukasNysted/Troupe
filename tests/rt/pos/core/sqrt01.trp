let fun f x n =     
  if n > 0 then 
  	let val  y = sqrt x in 
	print y;
	f y (n -1)
	end
  else () 
in f (65536*65536) 16;
   f (10000000000000000) 10
end 