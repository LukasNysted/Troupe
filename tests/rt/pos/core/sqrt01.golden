2024-02-19T10:01:21.535Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
65536
256
16
4
2
1.4142135623730951
1.189207115002721
1.0905077326652577
1.0442737824274138
1.0218971486541166
1.0108892860517005
1.0054299011128027
1.0027112750502025
1.0013547198921082
1.0006771306930664
1.0003385080526823
100000000
10000
100
10
3.1622776601683795
1.7782794100389228
1.333521432163324
1.1547819846894583
1.0746078283213176
1.036632928437698
>>> Main thread finished with value: ()@{}%{}
